### PR TITLE
Correct comment for quiche h3 connection

### PR DESCRIPTION
**Pull request by wangfuyu**

### CODE REVIEW

File: quiche/include/quiche.h [hunk 1]
```diff
@@ -786,7 +786,7 @@ void quiche_h3_config_enable_extended_connect(quiche_h3_config *config, bool ena
 // Frees the HTTP/3 config object.
 void quiche_h3_config_free(quiche_h3_config *config);
 
-// A QUIC connection.
+// A HTTP/3 connection.
```
Comment (ghedo) [inline quiche/include/quiche.h:789]:
```suggestion
// An HTTP/3 connection.
```

**Review by ghedo — APPROVED** [review]

**Comment by ghedo** [comment]
Merged, thanks!

File: quiche/include/quiche.h [hunk 2]
```diff
@@ -786,7 +786,7 @@ void quiche_h3_config_enable_extended_connect(quiche_h3_config *config, bool ena
 // Frees the HTTP/3 config object.
 void quiche_h3_config_free(quiche_h3_config *config);
 
-// A HTTP/3 connection.
+// An HTTP/3 connection.
```
Comment (wangfuyu) [inline quiche/include/quiche.h:789]: @ghedo why here use "An"? I think here must use "a", same with other comments.

Comment (ghedo) [reply quiche/include/quiche.h:789]: Because as far as I know that's how English works (see e.g. https://english.stackexchange.com/a/468483).

**Pull request merged**